### PR TITLE
Bug 1136089 - Bad markup in status bar quick settings

### PR DESCRIPTION
**Pull request by gmarty**

### CODE REVIEW

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=4caf0b6fab70)

File: apps/system/index.html [hunk 1]
```diff
@@ -252,9 +252,6 @@
     <script defer src="shared/js/input_parser.js"></script>
     <script defer src="js/value_selector/trusted_ui_value_selector.js"></script>
 
-    <!-- Trusted UI -->
-    <link rel="stylesheet" type="text/css" href="style/trusted_ui/trusted_ui.css">
-
```
Comment (etiennesegonzac) [inline apps/system/index.html]: ? dead code ?

**Comment by try-server-hook** [comment]
[gmarty Guillaume C. Marty (gmarty)](https://github.com/gmarty) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0d3e739103d0)

File: apps/system/index.html (hunk 1 above)
Comment (gmarty) [inline apps/system/index.html]: Yep, this link tag has been around for a while now, but dead since long.

**Pull request merged**